code/core/src/components/components/tooltip/ListItem.tsx

```diff
@@ -199,11 +199,17 @@ const ListItem = ({
 }: ListItemProps) => {
   const commonProps = { active, disabled };
   const itemProps = getItemProps(onClick, href, LinkWrapper);
+  const Icon: React.JSXElementConstructor<any> =
```

Comment:
I updated the logic to render any icon that's passed forward. This works, but may want to be more restrictive and change logic to pass boolean only show a CheckIcon. Unless this component is mean to be able to display icons other than check.

Reply:
Looks like this change broke some things, working on resolving.